src-tauri/tauri.conf.json

```diff
@@ -38,7 +38,6 @@
         "providerShortName": null,
         "signingIdentity": null
       },
-      "resources": ["web"],
```

Comment:
I'm using this directory to house a statically exported react app at the moment (original streambrah for the widgets). Will readd this once I tie that stuff into this repository.